src/node_errors.cc

```diff
@@ -62,7 +62,7 @@ static std::string GetErrorSource(Isolate* isolate,
   // added in the JavaScript context:
   Environment* env = Environment::GetCurrent(isolate);
   const bool has_source_map_url =
-      !message->GetScriptOrigin().SourceMapUrl().IsEmpty();
+      !message->GetScriptOrigin().SourceMapUrl()->IsUndefined();
```

Comment:
If there is no source map url in the code, `SourceMapUrl()` is `"undefined"` and `SourceMapUrl().IsEmpty()` is `false`.